src/schema/v2/me/index.ts

```diff
@@ -125,6 +129,56 @@ export const meType = new GraphQLObjectType<any, ResolverContext>({
   interfaces: [NodeInterface],
   fields: {
     ...IDFields,
+    alert: {
+      type: AlertType,
+      args: {
+        id: {
+          type: new GraphQLNonNull(GraphQLString),
```

Comment:
Yea, I was noticing we didn't roll with uuid's for `UserSearchCriteria` (we _probably_ should have but no biggie).

I _think_ it might be safer to treat these as strings here, just to be a bit more 'standard' and general-purpose (imagine eventually switching to a uuid, or slugs are supported). That way a client can always do a `toString` on the `id` argument as a rule.